### PR TITLE
BAR-1233 Updating code examples

### PR DESCRIPTION
**Pull request by AntonStasyuk**
I deleted everything about changelogs because there are no mentions of them in the readme.md file. Сollected it with a separate commit [here](https://github.com/borzodelivery/backend-code-sample/pull/2/commits/303829be15664ac0a9ca0da7a1691db120e06897).

If I did it in vain, let me know, I'll roll back the changes.

### CODE REVIEW

**Comment by borzo-verter** [comment]
YouTrack: [BAR-1233](https://youtrack.borzo.team/issue/BAR-1233)

File: readme.md [hunk 1]
```diff
@@ -27,37 +27,37 @@
 ## Миграции
```
Comment (anton-kotik) [inline readme.md]: Let's also translate the readme file. Just like we translate such files in the handbook.

* There should be two files: readme.md (in English) and readme-ru.md (in Russian).
* Add links `🇷🇺 Switch to Russian` and `🇬🇧 Switch to English`

File: library/Dostavista/Features/CourierGreetings/CourierGreetingRow.php [hunk 2]
```diff
@@ -1,45 +1,17 @@
...
- * @property string $allowed_to_show_finish_time Допустимое время окончания показа приветствия. По умолчанию 23:59:59.
- * @property bool   $is_deleted                  Удалена ли запись.
+ * @property int    $courier_greeting_id         Courier Greeting ID.
+ * @property string $greeting_template           Template with greeting text
+ * @property string $allowed_to_show_start_time  Allowed to show interval start time. Default is 00:00:00.
+ * @property string $allowed_to_show_finish_time Allowed to show interval finish time. Default is 23:59:59.
+ * @property bool   $is_deleted                  Has the entry been deleted?
  */
 class CourierGreetingRow extends TableRowAbstract {
-    protected function postInsert(): void {
```
Comment (anton-kotik) [inline library/Dostavista/Features/CourierGreetings/CourierGreetingRow.php:20]: Need to revert the changelog removing.

File: readme.md [hunk 3]
```diff
@@ -1,33 +1,35 @@
 
 # Borzo Backend Code Sample
 
-Давайте по шагам разберём на простой задаче, как мы создаём новые фичи в Борзо (aka Достависте).
+[🇷🇺 Switch to Russian](https://github.com/borzodelivery/backend-code-sample/readme-ru.md)
 
-Код здесь только ради примера, его не получится запустить, но его можно почитать.
+Let's go step by step through a simple task, how we create new features in Borzo (aka Dostavista).
```
Comment (anton-kotik) [inline readme.md]: For companies afaik should be used “at” — “at Borzo”

**Pull request merged**